### ISSUE TITLE
"is implemented in both" runtime errors when integrating xcframeworks

### ISSUE THREAD

**Issue by arielelkin** [segmentio/analytics-swift#233]
I'm integrating Segment using the xcframeworks available in this repo's and in Sovran's Releases:
https://github.com/arielelkin/segment-xcframework-bug

The app builds fine, but it leads to the following runtime errors and crashes:

```
objc[12470]: Class _TtC6Sovran5Store is implemented in both /Users/me/Library/Developer/Xcode/DerivedData/HosterFive-dkbcyakfxdmsbufprlbwoasbierr/Build/Products/Debug/HosterFive.app/Contents/Frameworks/Sovran.framework/Versions/A/Sovran (0x100548610) and /Users/me/Library/Developer/Xcode/DerivedData/HosterFive-dkbcyakfxdmsbufprlbwoasbierr/Build/Products/Debug/HosterFive.app/Contents/Frameworks/Segment.framework/Versions/A/Segment (0x100c40400). One of the two will be used. Which one is undefined.

objc[12470]: Class _TtC6Sovran9Container is implemented in both /Users/me/Library/Developer/Xcode/DerivedData/HosterFive-dkbcyakfxdmsbufprlbwoasbierr/Build/Products/Debug/HosterFive.app/Contents/Frameworks/Sovran.framework/Versions/A/Sovran (0x100548730) and /Users/me/Library/Developer/Xcode/DerivedData/HosterFive-dkbcyakfxdmsbufprlbwoasbierr/Build/Products/Debug/HosterFive.app/Contents/Frameworks/Segment.framework/Versions/A/Segment (0x100c40520). One of the two will be used. Which one is undefined.

2023-05-22 16:51:04.050325+0200 HosterFive[12470:3802694] Inspection is enabled by default for process or parent application with 'com.apple.security.get-task-allow' entitlement linked against old SDK. Use `inspectable` API to enable inspection on newer SDKs.
```

**Comment by arielelkin** [comment]
@bsneed any updates?

**Comment by bsneed** [comment]
Yup, about to dive in.  Last one on my list before a release. :D

**Comment by bsneed** [comment]
Fixed in 1.4.6
